src/entities/ValueNode/index.ts

```diff
@@ -33,8 +34,14 @@ export class ValueNode<ValueType = unknown> {
   /**
    * Returns serialized data associated with this value node.
    */
-  public get serialized(): ValueType {
-    return this.#value;
+  public get serialized(): ValueSerialized<ValueType> {
+    let value = this.#value;
+
+    if (typeof value === 'object' && this.#value !== null) {
+      value = Object.assign({ $t: BlockChildType.Value }, value);
```

Comment:
maybe it would be useful to move "$t" to some constant with clear name like "NODE_TYPE_HIDDEN_PROP"?